src/js/languages/default.json

```diff
@@ -44,9 +44,12 @@
     "open": "Open sort"
   },
   "dataSummary": {
-    "filtered": "{filteredTotal} results of {total} items",
-    "filteredSingle": "{filteredTotal} result of {total} items",
-    "total": "{total} items"
+    "filtered": "{filteredTotal} results of {total} {items}",
```

Comment:
It always seems weird to me these don't say `{filteredTotal} results out of {total} {items}"` That is just my opinion though.

Reply:
I can't fully remember all of the design decisions re: wording, except that it potentially was leaning towards brevity. Let's leave as is for now.